components/google_recaptcha/README.md

```diff
@@ -13,3 +13,5 @@ Here are some examples of what you can build using the Google reCAPTCHA API:
 - A simple CAPTCHA form that can be used to protect your website from spam
 - An advanced CAPTCHA form that can be used to protect your website from more
   sophisticated attacks
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).